index.d.ts

```diff
@@ -106,6 +106,16 @@ export type Options = {
 } & Readonly<GlobOptions> & CpFileOptions; // eslint-disable-line @typescript-eslint/no-redundant-type-constituents
 
 export type ProgressData = {
+	/**
+	Absolute path to source.
+	*/
+	sourcePath: string;
+
+	/**
+	Absolute path to destination.
+	*/
+	destinationPath: string;
```

Comment:
Add these to the bottom.

The docs should also make it clear that they are the **current** file.

And you also need to document (as in description) these in the readme.

Reply:
feel free to correct the document comment. Thanks